components/popover/popover-mixin.js

```diff
@@ -62,34 +69,107 @@ export const PopoverMixin = superclass => class extends superclass {
 
 	constructor() {
 		super();
+		this.noAutoClose = false;
 		this.opened = false;
-		this._useNativePopover = isSupported;
+		this._useNativePopover = isSupported ? 'manual' : undefined;
```

Comment:
The `_useNativePopover` type was changed to `String` since we need to set it to `manual`. We don't want the `auto` popover behaviour because a) we want to provide consistent auto-close behaviour for native and non-native popover, and b) we will want to handle the `[Escape]` key with our own [dismissible helper](https://github.com/BrightspaceUI/core/blob/main/helpers/dismissible.js).